<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakByFunctionDialog</class>
 <widget class="QDialog" name="BreakByFunctionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Breakpoint at Function</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="functionLabel">
       <property name="text">
        <string>Function to break on:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="functionLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
